src/services/analytics/events/push-notifications.ts

```diff
@@ -38,7 +38,7 @@ export const PUSH_NOTIFICATION_EVENTS = {
   },
   // User enabled all notifications from banner
   ENABLE_ALL: {
-    action: 'Enable all notifications',
+    action: 'Enable all notifications on chain',
```

Comment:
Let's not change it now, otherwise we'll end up with orphan events. Also the action should be short.